docs/it/authorization_endpoint.rst

```diff
@@ -61,67 +65,96 @@ L’oggetto request DEVE essere un token JWT firmato, secondo le modalità defin
+       Quindi chiede il consenso al trasferimento degli attributi. 
+
      - |check-icon|
    * - **redirect_uri**
-     - URL dove l’OP reindirizzerà l’utente al termine del processo di autenticazione.
-     - Deve essere uno degli URL indicati nel client metadata (v. paragrafo 3.2). 
+     - URL dove l'OP reindirizzerà l'utente al termine del processo di autenticazione. 
+       Deve essere uno degli URL indicati nel client metadata (v. paragrafo 3.2). 
      - |check-icon|
    * - **response_type**
-     - Il tipo di credenziali che deve restituire l’OP.
-     - **code**
+     - Il tipo di credenziali che deve restituire l'OP.
+       **code**
      - |check-icon|
    * - **scope**
      - Lista degli scope richiesti.
-     - **openid** (obbligatorio).
+       
+       **openid**: (obbligatorio).
+       
+       **offline_access**: se specificato, l'OP rilascerà oltre all'access token anche un refresh token necessario
+       per instaurare sessioni lunghe revocabili. L'uso di questo valore è consentito solo se se si intende offrire all'utente una `sessione lunga revocabile <https://www.agid.gov.it/sites/default/files/repository_files/spid-avviso-n41-integrazione_ll.gg_._openid_connect_in_spid.pdf#page=6>`_.
 
-       **offline_access**: se specificato, l’OP rilascerà oltre all’access token anche un refresh token necessario per instaurare sessioni lunghe revocabili. L’uso di questo valore è consentito solo se se si intende offrire all’utente una `sessione lunga revocabile <https://www.agid.gov.it/sites/default/files/repository_files/spid-avviso-n41-integrazione_ll.gg_._openid_connect_in_spid.pdf#page=6>`_.
      - |check-icon|
    * - **acr_values**
-     - Valori di riferimento della classe di contesto dell’autenticazion e richiesta. Stringa separata da uno spazio, che specifica i valori “acr” richiesti al server di autorizzazione per l’elaborazione della richiesta di autenticazione, con i valori visualizzati in ordine di preferenza. L’OP ha facoltà di utilizzare un’autenticazione ad un livello più alto di quanto richiesto. Tale scelta non deve comportare un esito negativo della richiesta.
-     - https://www.spid.gov.it/SpidL1
-       https://www.spid.gov.it/SpidL2
-       https://www.spid.gov.it/SpidL3
+     - Valori di riferimento della classe di contesto dell'Authentication Request. 
+       String separata da uno spazio, che specifica i valori "acr" richiesti al server di autorizzazione per l'elaborazione della richiesta di autenticazione con i valori indicati in ordine di preferenza. L'OP può utilizzare un'autenticazione ad un livello più alto di quanto richiesto. Tale scelta non deve comportare un esito negativo della richiesta.
+       Deve contenere per SPID uno o più valori tra i seguenti:
+       
+       ``https://www.spid.gov.it/SpidL1``
+       ``https://www.spid.gov.it/SpidL2``
+       ``https://www.spid.gov.it/SpidL3``
+
+       Deve contenere per CIE uno o più valori tra i seguenti:
+       
+       ``CIE_L1`` |br|
```

Comment:
CIE si è uniformata a SPID. Gli **acr_values** sono i medesimi
@sp640919 @peppelinux

Reply:
Uniformato tutto a:        
       ``https://www.spid.gov.it/SpidL1``
       ``https://www.spid.gov.it/SpidL2``
       ``https://www.spid.gov.it/SpidL3``